helper/sshutil/session_pool.go

```diff
@@ -151,11 +154,11 @@ func (p *pool) getConn(k, addr string, config *ssh.ClientConfig) *conn {
 	if c.err == nil {
 		// Connection succeeded
 		c.name = fmt.Sprintf("%s-%s-%x", addr, config.User, c.c.SessionID())
-		metrics.IncrCounter(metricsutil.CleanupMetricKey([]string{"ssh-connections-pool", "creations", c.name}), 1)
+		metrics.IncrCounter(metricsutil.CleanupMetricKey([]string{"ssh-connections-pool", "creations"}), 1)
```

Comment:
No interest in having connection name in labels here?

Reply:
Here we measure the number of created ssh connections. Its not an information about a particular connection, that's why connection name is not pertinent

Reply:
ok